From 92b1d0d2ea8cc87bf6a9b9084aa76ca3580d5e4e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Thu, 19 Aug 2021 09:26:41 +0200
Subject: [PATCH] configure.ac: Fix compatibility with dash shell

Fix the compatibility of the configure script with the dash shell
by replacing the use of bash-specific '==' operator with plain '='.
The calls in question do not utilize the pattern-matching behavior
of '=='.
---
 configure.ac | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index c872a9b3..283c90e2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -709,7 +709,7 @@ AC_DEFUN([A8_OPTION],[
     fi
 ])
 
-if [[ "$a8_target" == "libatari800" ]]; then
+if [[ "$a8_target" = "libatari800" ]]; then
     WANT_NEW_CYCLE_EXACT=yes
     WANT_VERY_SLOW=no
     WANT_CRASH_MENU=no
@@ -1040,7 +1040,7 @@ if [[ "$with_sound" != no ]]; then
               VOICEBOX,[Define to emulate the Alien Group Voice Box.]
              )
 
-    if [[ "$with_sound" == "libatari800" ]]; then
+    if [[ "$with_sound" = "libatari800" ]]; then
         WANT_SOUND_CALLBACK=no
         WANT_CONSOLE_SOUND=yes
         WANT_SERIO_SOUND=yes
@@ -1094,7 +1094,7 @@ A8_OPTION(pokeyrec,$WANT_POKEYREC,
           [Provide Pokey registers recording (default=ON)],
           POKEYREC,[Define to add Pokey registers recording.]
          )
-if [[ "$WANT_POKEYREC" == "yes" ]]; then
+if [[ "$WANT_POKEYREC" = "yes" ]]; then
     AC_SYS_LARGEFILE
 fi
 AM_CONDITIONAL([WANT_POKEYREC], test "$WANT_POKEYREC" = "yes")
-- 
2.33.0

